parser_library/src/workspaces/workspace_configuration.cpp

```diff
@@ -311,6 +314,8 @@ void workspace_configuration::process_processor_group_and_cleanup_libraries(
     for (const auto& pg : pgs)
         process_processor_group(pg, fallback_macro_extensions, alternative_root, diags);
 
+    m_proc_grps.try_emplace(std::make_pair("*NOPROC*", utils::resource::resource_location()), m_implicit_proc_group);
+
```

Comment:
This will have impact on the various `&grp == &implicit_proc_grp` tests.
I suspect it might be more work, but wouldn't just returning `nullptr` from `get_program` accomplish the task? The program structure could e.g. contain optional proc_grp_id.

Reply:
I'll look into that. There's also some issue with b4g => converting this back to draft